avg_temp_job.txt

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
FYI http://robertmuth.blogspot.com/2012/08/better-bash-scripting-in-15-minutes.html